internal/executor/util/kubernetes_object.go

```diff
@@ -89,6 +94,20 @@ func CreateIngress(name string, job *api.Job, pod *v1.Pod, service *v1.Service,
 		rules = append(rules, path)
 	}
 
+	tls := make([]networking.IngressTLS, 0, 1)
```

Comment:
Should there be a test for this new logic in `kubernetes_objects_test,go`?

Arguably we should also have a test for the changes to `ingres_util`, but I could accept the argument that maybe the code is trivial enough that we don't need to test this?

Reply:
I'll have a look at adding a test, but what logic in specific do you mean when you say this and highlight line 97?

Reply:
I mean we should have a test for all the new logic you've added in this function.  I.e. that if we pass a jobConfig with  `TlsEnabled = true` into the function, we get back an ingress with the correct `IngressSpec` set.

Reply:
we don't test this function at all currently. I'm not entirely sure what the value of this is because generating this ingress spec by hand seems like an exercise in futility considering there might be multiple ways to represent the same fundamental k8s ingress in go structs. So I fear to actually generate this config we'd have to either use the function (which isn't really a test at that point) or follow the function through by hand which isn't any better (arguably worse)